### DISCUSSION TITLE
what exactly are the differences between the free and full paid version?

### DISCUSSION THREAD

**Discussion by nutpantz** [Hamza417/Positional#129] [Q&A]
what exactly are the differences between the free and full paid version?

**Comment by Hamza417** [comment]
It's mentioned [here](https://github.com/Hamza417/Positional/blob/7f9bb067a6eab75b2a28c52d32c70a4ba8f2b3d2/app/src/main/assets/html/buy_full.html#L18C1-L35C6) and in the app as well. These features are exclusive to paid versions only.

<ul>
    <li>Bloom effect in compass.</li>
    <li>Various bloom styles.</li>
    <li>Clock needle skins.</li>
    <li>Custom location mode.</li>
    <li>Location pin skins.</li>
    <li>World, UTC and Zulu times using custom time zones.</li>
    <li>Beautiful set of launcher app icons.</li>
    <li>Ability to customize pin style in location panel</li>
    <li>More than 80 splash screen arts.</li>
    <li>Change app level coordinates format settings.</li>
    <li>Sun and moon position's visual representation in <b>Time</b> panel.</li>
    <li>Material You accent color in Accent Colors.</li>
    <li>Modify corner radius of the app to your liking.</li>
    <li>Ability to change motion style of the clock needle.</li>
    <li>Modify compass needle motion style dynamically to suit your needs.</li>
</ul>
